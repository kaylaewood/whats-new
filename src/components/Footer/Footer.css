.footer {
  grid-area: footer;
  /* display: flex;
  justify-content: center; */
  margin-bottom: 25px;
}
