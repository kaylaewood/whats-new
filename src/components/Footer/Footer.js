import React from 'react'
import './Footer.css'

function Footer() {
  return (
    <div className='footer'>
      Icons from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a>
    </div>
  )
}

export default Footer;
