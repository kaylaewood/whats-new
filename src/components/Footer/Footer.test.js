import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer';
import { shallow } from 'enzyme';

describe('Footer', () => {
  it('should correctly match the snapshot with all the data passed correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  })
});
