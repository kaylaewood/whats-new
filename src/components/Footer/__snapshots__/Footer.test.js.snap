// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should correctly match the snapshot with all the data passed correctly 1`] = `
<div
  className="footer"
>
  Icons from 
  <a
    href="https://www.flaticon.com/"
    title="Flaticon"
  >
    www.flaticon.com
  </a>
</div>
`;
